### DISCUSSION TITLE
How to convert NotionDate object to Python datetime?

### DISCUSSION THREAD

**Discussion by ghost** [jamalex/notion-py#288] [Q&A]
Hello There! I've described my problem in [this issue](https://github.com/jamalex/notion-py/issues/286), but, unfortunately, nobody answered. Please, help me with this task.

**Comment by Mtigo** [comment]
Perhaps the problem is this: notion.collection.NotionDate, unlike the usual date format, contains 2 properties: _start_ and _end_. If a regular date (d) can be converted, such as d.strftime("%Y-%m-%d"), then notion.collection.NotionDate (nd )needs to convert either nd._**start**_.strftime ("%Y-%m-%d") or nd._**end**_.strftime ("%Y-%m-%d")
